snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: plexmediaserver
-version: 1.2.2.2857-d34b464-1
+version: 1.2.2.2857-d34b464
```

Comment:
Is it really needed just for packaging? Feels like it comes from the packaging world, very old school.

If it is needed, then we need to duplicate the version entry in the download link which doesn't feel right either.